docs/release-guide.md

```diff
@@ -174,44 +174,44 @@ $ cd apache-apisix-1.0 && ./utils/check-version.sh 1.0 && cd ..
 $ rm -rf apache-apisix-1.0/.git
 
 # make tar package
-$ tar zcvf apache-apisix-1.0-rc1-src.tar.gz apache-apisix-1.0
+$ tar zcvf apache-apisix-1.0-src.tar.gz apache-apisix-1.0
```

Comment:
we already has `makefile` to do this job, don't need this anymore

Reply:
Done.